java/src/jmri/jmrix/JmrixSystemsBundle_es.properties

```diff
@@ -1,30 +1,54 @@
-# JmritSystemsBundle_es.properties
+# java/src//jmri/jmrix/JmrixSystemsBundle_es.properties
+# JmrixSystemsBundle.properties
 #
-# Spanish properties for the jmri.jmrix Systems menu
-# Translated by Alfredo Sola <alfredos@users.sf.net> 20040826
+# Default properties for the jmri.jmrix Systems menu
```

Comment:
Is this intended? Removed information about who previously translated the file makes me wonder.

There are more cases in this PR of removed notes about previously translation.